src/command/render/latexmk/texlive.ts

```diff
@@ -285,14 +286,33 @@ function tlmgrCommand(
   args: string[],
   _quiet?: boolean,
 ) {
+  let cmdExec: string[];
+  let tempFile: string | undefined;
+  if (Deno.build.os === "windows") {
+    // writing to a file to exectute to avoid quoting issue with Deno
+    // https://github.com/quarto-dev/quarto-cli/issues/336
+    const lines = ["tlmgr", cmd, ...args];
+    tempFile = Deno.makeTempFileSync(
+      { prefix: "tlmgr-cmd", suffix: ".bat" },
+    );
+    Deno.writeTextFileSync(tempFile, lines.join(" ") + "\n");
```

Comment:
I looked into `temp.ts` but was not sure any function should have been used here. Temp file is written in default temp dir. Maybe a special quarto-session dir should be used instead ? Not sure. 
I used default Deno temp file then.